core/store/models/common.go

```diff
@@ -514,7 +402,7 @@ func (s Sha256Hash) Value() (driver.Value, error) {
 }
 
 // URL extends url.URL to implement encoding.TextMarshaler.
```

Comment:
should this and the follow 2 funcs have a `Deprecated` comment (example https://github.com/smartcontractkit/chainlink/blob/09067a354b0c6145511b584f3330eacd390aeb7e/core/utils/utils.go#L146) ?

Reply:
Desirable if possible, although now that the linter is working again, it force you to update all uses at the same time.